### PR TITLE
[Robot] Enable robot tests for non admin users

### PR DESCRIPTION
**Pull request by rmonica08**
1. Enable test for non admin users
2. Added test for View Perm set: Add More Service Participants quick action

# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

# Definition of Done
  Refer to [Asteroids DoD document](https://salesforce.quip.com/iq2mAy4i62oM) to see any additional details for the items below
- [ ] Any net new LWC work has Sa11y tests & 50% or above lines JEST test coverage  
- [ ] CRUD/FLS is enforced in Apex
- [ ] Permission sets are updated to account for CRUD|FLS|Tab|Classes
- [ ] Field sets are updated to account for new fields
- [ ] UX approval or UX not necessary
- [ ] Link the pull request and work item by PR comment and Chatter post respectively, e.g. GUS: W-0000000: Work Name (https://github.com/SalesforceFoundation/PMM/pull/303)
- [ ] All **acceptance criteria** have been met
    - [ ] Developer
    - [ ] Code Reviewer
    - [ ] QA
- [ ] PR contains draft release notes
- [ ] QE story level testing completed

### CODE REVIEW

File: robot/pmm/tests/browser/Program/add_service.robot [hunk 1]
```diff
@@ -26,6 +29,7 @@ Setup Test Data
 *** Test Cases ***
 
 Create Service from Program Object
```
Comment (kselvocki) [inline robot/pmm/tests/browser/Program/add_service.robot:31]: Add Documentation for this test case

File: robot/pmm/tests/browser/Program/pe_quickaction.robot [hunk 2]
```diff
@@ -36,6 +38,7 @@ Setup Test Data
 
 *** Test Cases ***
 Add contact to program quick action on Program
```
Comment (kselvocki) [inline robot/pmm/tests/browser/Program/pe_quickaction.robot:40]: Add Documentation for this test case

File: robot/pmm/tests/browser/Service/new_service.robot [hunk 3]
```diff
@@ -25,6 +28,7 @@ Setup Test Data
 
 *** Test Cases ***
 Create Service from top nav
```
Comment (kselvocki) [inline robot/pmm/tests/browser/Service/new_service.robot:30]: Add Documentation and feature:Service tag

File: robot/pmm/tests/browser/ServiceDeliveries/new_service_delivery.robot [hunk 4]
```diff
@@ -36,16 +38,17 @@ Setup Test Data
 Create a Service Delivery via UI
     [Documentation]                        This test creates Service Delivery record and verifies that the Service Delivery record
     ...                                    has all the values from the form
+    [tags]                                 perm:admin   perm:manage     perm:deliver     feature:Service Deliver
```
Comment (kselvocki) [inline robot/pmm/tests/browser/ServiceDeliveries/new_service_delivery.robot:41]: Fix tag feature:Service Delivery

**Review by kselvocki — COMMENTED** [review]
Just a few documentation and tag updates. Otherwise, looks awesome Monica!

**Review by gaganpsandhu — APPROVED** [review]

File: robot/pmm/tests/browser/ServiceDeliveries/new_service_delivery.robot (hunk 4 above)
Comment (rmonica08) [reply robot/pmm/tests/browser/ServiceDeliveries/new_service_delivery.robot]: Fixed

File: robot/pmm/tests/browser/Service/new_service.robot (hunk 3 above)
Comment (rmonica08) [reply robot/pmm/tests/browser/Service/new_service.robot]: Done

File: robot/pmm/tests/browser/Program/pe_quickaction.robot (hunk 2 above)
Comment (rmonica08) [reply robot/pmm/tests/browser/Program/pe_quickaction.robot:40]: Fixed

File: robot/pmm/tests/browser/Program/add_service.robot (hunk 1 above)
Comment (rmonica08) [reply robot/pmm/tests/browser/Program/add_service.robot:31]: Fixed

**Review by kselvocki — APPROVED** [review]

**Pull request merged**